content/en/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Feedback Appreciated
+category: technology
+tags: ["fundamental", "", ""]
+---
+
+## What it is
+
+An Ingress is an abstraction that helps to manage the traffic from the outside into a [Kubernetes](/kubernetes/) cluster. 
+It consists of two elements, the ingress resource, and the ingress controller. 
+The ingress resource is just a Kubernetes manifest file that lives along with other manifest files and allows for the configuration of traffic routing. 
+The ingress controller is the web server technology that actually performs the routing of the traffic according to the configuration in the ingress resource.
+
+## Problem it addresses
+
+Traditionally, users access deployed production-grade applications using URLs, typically HTTPS, but whenever the traffic increases, it leads to the need for an increase in the number of replicas of the pod to handle it. The pod is the smallest unit of workload in Kubernetes. It is the object that contains the application. A load balancer is present between the application and the users to handle traffic for a specific service.
```

Comment:
```suggestion

```
My suggestion is to delete this completely as is does not really fit to the rest and it is a bit missleading.